### DISCUSSION TITLE
urql for svelte

### DISCUSSION THREAD

**Discussion by aditya-padhi-kbl** [urql-graphql/urql#1111] [Q&A] [answered]
Can you please share some documentation regarding urql integration with Svelte ?

**Comment by frederikhors** [comment, marked as answer]
1. https://formidable.com/open-source/urql/docs/api/svelte/
2. https://github.com/FormidableLabs/urql/tree/main/packages/svelte-urql/example

**Reply by kitten** [reply]
Also on a side-note, apart from the API docs, all relevant documentation on our site has a "Svelte" section as well https://formidable.com/open-source/urql/docs/basics/

**Reply by aditya-padhi-kbl** [reply]
Thank-you very much :)
